recipes/mmtf-python/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "mmtf-python" %}
+{% set version = "1.0.4" %}
+{% set sha256 = "8aa92bc66b055f952c88f2dc33cd5dd52e39df376bd8ad88cca1cc344b2990da" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - msgpack-python
```

Comment:
Sometime it is imported in the `setup.py` and, even though `--single-version-externally-managed` is enable, we still need to list the dependencies in the `build` section. @kain88-de can you check that?